File: wavearray/wave.java

	
	public static void main(String[] args)
    {
			Scanner sc=new Scanner(System.in);
            int n = sc.nextInt(); 
			int []arr= Arrays.stream(sc.next().split("[,\\[\\]]")).filter(w-> !w.equals("")).mapToInt(Integer::parseInt).toArray();
		    Solution g=new Solution();
            int []ans=g.wave_array(arr,n);  
            for(int i=0;i<n;i++)
            System.out.print(ans[i] + " ");  
          
	}
}
class Solution
{
	public static int[] wave_array(int arr[],int n)
        {
            int[] lex_smallarray = new int[n];
			///Write your code here without removing the exising code
	//the variable 'arr' contains array of integers of size n.
	//modified the array 'lex_smallarray' contains the output of the program.
			return lex_smallarray;
        }
}

File: wavearray/wave.py
# Starter Code
class Solution:
    def wave_array(self, arr, n):
        lex_smallarray=[0]
        # Write your code here without removing the exising code
        # the variable 'arr' contains array of integers of size n.
        # modified the array 'lex_smallarray' contains the output of the program.
        return lex_smallarray


if __name__ == '__main__':
    n = int(input())
    str = input()
    arr = [int(int(i)) for i in str.strip('[]').split(',')]
    x = Solution()
    ans=x.wave_array(arr, n)
    for i in range(0, len(ans)):
        print(ans[i], end=" ")
